example1/src/tests/integration/product.integration.test.ts

```diff
@@ -0,0 +1,107 @@
+import { MikroORM, RequestContext } from "@mikro-orm/core";
+import { PostgreSqlDriver } from "@mikro-orm/postgresql";
+import { StatusCode } from "@panenco/papi";
+import { expect } from "chai";
+import supertest from "supertest";
+import { App } from "../../app";
+import { FridgeBody } from "../../contracts/fridge.body";
+import { LoginBody } from "../../contracts/login.body";
+import { ProductBody } from "../../contracts/product/product.body";
+import { UserBody } from "../../contracts/user.body";
+import { User } from "../../entities/user.entity";
+
+describe("Integration tests", () => {
```

Comment:
Refactor naar `Integration tests product`?
Anders ga je twee dezelfde te zien krijgen in die extension view in code